akka-http-bench-jmh/src/main/scala/akka/http/impl/engine/http2/H2ServerProcessingBenchmark.scala

```diff
@@ -22,6 +22,10 @@ import akka.util.ByteString
 import com.typesafe.config.ConfigFactory
 import org.openjdk.jmh.annotations._
 
+/**
+ * Emulates request bytes coming in, selecting HTTP/2 over HTTP/1, sending a static response back,
+ * validating the response bytes come out again.
+ */
```

Comment:
In my branch, I added more params / features to this benchmark to avoid measuring the client as well. I'm fine with also adding the client/server version.

Reply:
Ah, cool, good to have both probably (but for this particular case indeed better to avoid measuring the client as well)